### PR TITLE
Register and upgrade CaaSP after the deployment

### PR DESCRIPTION
**Pull request by jsuchome**
Based on the official guidelines from the CaaSP team. 

Alternative to https://github.com/SUSE-Cloud/socok8s/pull/164

### CODE REVIEW

File: playbooks/openstack-prepare_caasp_workers_for_update.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    - name: Pre-flight checks for the role
+      assert:
+        that:
+          - caasp_registration_key is defined
+
+    - name: Register node in SCC
+      command: "SUSEConnect -r {{ caasp_registration_key }}"
+
+    - name: Disable the transactional-update timer
+      command: "systemctl disable --now transactional-update.timer"
```
Comment (Itxaka) [inline playbooks/openstack-prepare_caasp_workers_for_update.yml]: should we re-enable it afterwards? Is there a reason to disable it?

File: playbooks/openstack-update_caasp.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    - name: Load variables
+      include_vars: "{{ item }}"
+      loop:
+        - "{{ playbook_dir }}/../vars/common-vars.yml"
+        - "{{ playbook_dir }}/../vars/manifest.yml"
+      tags:
+        - always
+
+    - name: Use Velum tooling to update CaaSP admin node
+      shell: "./velum-interactions --update-admin --update-minions -e {{ upstream_repos_clone_folder }}/ccp/environment.json"
```
Comment (Itxaka) [inline playbooks/openstack-update_caasp.yml]: do we need update-minions here? shouldnt that be update-admin only?

Comment (jsuchome) [reply playbooks/openstack-update_caasp.yml]: I splitted it in to tasks in updated commit

File: playbooks/openstack-prepare_caasp_workers_for_update.yml (hunk 1 above)
Comment (jsuchome) [reply playbooks/openstack-prepare_caasp_workers_for_update.yml]: TBH I have no idea

**Comment by Itxaka** [comment]
@jsuchome this needs a rebase? or are you gonna do a different approach?

**Comment by jsuchome** [comment]
Closing, we may not need it

**Pull request closed without merging**